index.yml

```diff
@@ -387,3 +387,13 @@ Projects:
     target-file  : Dockerfile
     notify-email : mohammed.zee1000@gmail.com
     depends-on   : null
+
+  - id          : 520
+    app-id      : gluster
+    job-id      : storagesig-gluster-centos
+    git-url     : https://github.com/gluster/docker
+    git-path    : CentOS/
+    git-branch  : master
+    target-file : Dockerfile
```

Comment:
Sure. Thanks @kbsingh  . I feel,  it can follow in a different PR.  For now, lets add master index.